understanding/20/error-identification.html

```diff
@@ -1,124 +1,104 @@
 		<dl>
          <dt>Goal</dt><dd>Users know an error exists and what is wrong.</dd>
          <dt>What to do</dt><dd>Provide descriptive notification of errors.</dd>
-         <dt>Why it's important</dt><dd>Flagging errors helps people with reduced sight and cognitive disabilities resolve them.</dd> 
+         <dt>Why it's important</dt><dd>Flagging errors helps people with reduced sight and cognitive disabilities resolve them.</dd>
       </dl>
 
    </section>
-   
+
    <section id="intent">
       <h2>Intent of Error Identification</h2>
-      
-      
-      <p>The intent of this Success Criterion is to ensure that users are aware that an error
-         has occurred and can determine what is wrong. The error message should be as specific
-         as possible. 
-         In the case of an unsuccessful form submission, re-displaying the form and indicating
-         the fields in error is insufficient for some users to perceive that an error has occurred.
-         Screen reader users, for example, will not know there was an error until they encounter
-         one of the indicators. They may abandon the form altogether before encountering the
-         error indicator, thinking that the page simply is not functional. Per the definition,
-         an "input error" is information provided by the user that is not accepted. This includes: 
+
+      <p>The intent of this success criterion is to ensure that users are aware that an error
+         has occurred and can determine what is wrong. In the case of an unsuccessful form submission,
+         it is not sufficient to only re-display the form without providing any hint that the submission
+         failed.
+         The error can be indicated in <a>text</a>, which does allow for visually apparent images or
+         icons with suitable alternative text.
+         Whether or not an error message provides users with sufficient information about the nature of
+         the error, and what they should do to correct it, is covered more specifically by
+         <a href="error-suggestion">3.3.3 Error Suggestion</a>.
       </p>
-      
+      <p>An "input error" is information provided by the user that is not accepted. This includes:</p>
+
       <ul>
-         
          <li>information that is required by the web page but omitted by the user, or</li>
-         
-         <li>information that is provided by the user but that falls outside the required data
-            format or allowed values.
-         </li>
-         
+         <li>information that is provided by the user but that falls outside the required data format or allowed values.</li>
       </ul>
-      
+
       <p>For example:</p>
-      
+
       <ul>
-         
-         <li>the user fails to enter the proper abbreviation in to state, province, region, etc.
-            field; 
-         </li>
-         
+         <li>the user fails to enter the proper abbreviation in to state, province, region, etc. field;</li>
```

Comment:
"to" seems wrong, and I've pulled out the "etc" as superfluous
```suggestion
         <li>the user fails to enter the proper abbreviation in a state, province, or region field;</li>
```